### ISSUE TITLE
Are these really ISO 639-1?

### ISSUE THREAD

**Issue by lazarljubenovic** [umpirsky/language-list#12]
The Readme mentions that the list is ISO 693-1. [According to ISO](https://www.iso.org/iso-639-language-codes.html), it "provides a **2 letter code** that has been designed to represent most of the major languages of the world". However, this repository contains **three-letter codes** as well. At a quick glance, it seems like these are a combination of 693-1 when available, and then falls back to 693-2 or 693-3. Is this the case? Would love to see this information stated in the readme as well.

**Comment by umpirsky** [comment]
Yes, I'm also curious what 3 letter codes there are, I guess 693-2 or 693-3.

**Comment by eighthave** [comment]
What @lazarljubenovic describes sounds right to me.  Though perhaps not strictly following a given standard, it is pretty common practice.

**Comment by fufkgjcinog-cmyk** [comment]
> The Readme mentions that the list is ISO 693-1. [According to ISO](https://www.iso.org/iso-639-language-codes.html), it "provides a **2 letter code** that has been designed to represent most of the major languages of the world". However, this repository contains **three-letter codes** as well. At a quick glance, it seems like these are a combination of 693-1 when available, and then falls back to 693-2 or 693-3. Is this the case? Would love to see this information stated in the readme as well.

**Comment by fufkgjcinog-cmyk** [comment]
[xiaomiPrivacyPolicy.pdf](https://github.com/user-attachments/files/23303171/xiaomiPrivacyPolicy.pdf)

**Comment by fufkgjcinog-cmyk** [comment]
@

**Comment by fufkgjcinog-cmyk** [comment]
### https://github.com/umpirsky/language-list/issues/12
